src/exchange/account/order/exchange-account-order.html

```diff
@@ -0,0 +1,124 @@
+                                    class="oui-radio__input"
+                                    type="radio"
+                                    name="renewalPeriod"
+                                    id="renewalPeriodP1M"
+                                    value="P1M"
+                                    checked="checked"
+                                    data-ng-model="$ctrl.renewalPeriod"
+                                    data-ng-change="$ctrl.updateValueOfServiceToTargetExpiresBeforeEndOfRenewalPeriod()">
+                                <label
+                                    class="oui-radio__label-container"
+                                    for="renewalPeriodP1M">
+                                    <span
+                                        class="oui-radio__label"
+                                        data-translate="exchange_account_order_renewalPeriod_P1M"></span>
+                                </label>
+                            </div>
+                            <div
+                                class="oui-field__helper"
+                                data-translate="exchange_account_order_renewalPeriod_P1M_description"
+                                data-translate-values="{
+                                    price: $ctrl.offer.prices.P1M.valueToDisplay,
+                                }"></div>
+                        </div>
+
+                        <div class="oui-field__control">
+                            <div class="oui-radio">
+                                <input
+                                    class="oui-radio__input"
+                                    type="radio"
+                                    name="renewalPeriod"
+                                    id="renewalPeriodP1Y"
+                                    value="P1Y"
+                                    checked="checked"
+                                    data-ng-model="$ctrl.renewalPeriod"
+                                    data-ng-change="$ctrl.updateValueOfServiceToTargetExpiresBeforeEndOfRenewalPeriod()">
+                                <label
+                                    class="oui-radio__label-container"
+                                    for="renewalPeriodP1Y">
+                                    <span
+                                        class="oui-radio__label"
+                                        data-translate="exchange_account_order_renewalPeriod_P1Y"></span>
+                                </label>
+                            </div>
+                            <div
+                                class="oui-field__helper"
+                                data-translate="exchange_account_order_renewalPeriod_P1Y_description"
+                                data-translate-values="{
+                                    price: $ctrl.offer.prices.P1Y.valueToDisplay,
+                                }"></div>
+                        </div>
```

Comment:
Didn't a `oui-field` with  nested `oui-radio` fit the needs ?

Reply:
No, the `onChanges` callback was called at the wrong time, I'll have to look into that